genesis-programs/src/lib.rs

```diff
@@ -16,28 +16,40 @@ use log::*;
 use solana_runtime::bank::{Bank, EnteredEpochCallback};
 
 pub fn get_inflation(operating_mode: OperatingMode, epoch: Epoch) -> Option<Inflation> {
-    let past_epoch_inflation = get_inflation_for_epoch(operating_mode, epoch.saturating_sub(1));
-    let epoch_inflation = get_inflation_for_epoch(operating_mode, epoch);
-
-    if epoch_inflation != past_epoch_inflation || epoch == 0 {
-        Some(epoch_inflation)
-    } else {
-        None
-    }
-}
-
-pub fn get_inflation_for_epoch(operating_mode: OperatingMode, epoch: Epoch) -> Inflation {
     match operating_mode {
-        OperatingMode::Development => Inflation::default(),
-        OperatingMode::Stable | OperatingMode::Preview => {
-            if epoch == std::u64::MAX {
+        OperatingMode::Development => {
+            if epoch == 0 {
+                Some(Inflation::default())
+            } else {
+                None
+            }
+        }
+        OperatingMode::Preview => {
+            if epoch == 0 {
+                // No inflation at epoch 0
+                Some(Inflation::new_disabled())
+            } else if epoch == 44 {
+                // testnet enabled inflation at epoch 44:
+                // https://github.com/solana-labs/solana/blob/d8e885f4259e6c7db420cce513cb34ebf961073d
+                Some(Inflation::default())
```

Comment:
I'm adding this mainly for cosmetic purpose because otherwise the code looks strange (only `new_disabled`/`None` in the code => when exactly inflation can be enabled??)